packages/portal/src/plugins/europeana/iiif/presentation.js

```diff
@@ -1,3 +1,7 @@
+import EuropeanaApi from '../apis/utils/base.js';
 import { BASE_URL as EUROPEANA_IIIF_BASE_URL } from './index.js';
```

Comment:
this isn't loading a BASE_URL

Reply:
No, it's importing the base class that the others extend.